deployment/api-deployment.yaml

```diff
@@ -23,9 +23,8 @@ spec:
     spec:
       containers:
         - args:
-            - monocle-api
-            - --port
-            - "9898"
+            - monocle
+            - api
```

Comment:
Looks like MONOCLE_API_PORT is missing from the env and we rely on the default in Monocle.CLI.

Reply:
I think that's ok because this is in kubernetes internal network.